modules/launcher/src/main/java/org/mule/module/launcher/descriptor/DomainDescriptor.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.launcher.descriptor;
+
+public class DomainDescriptor extends ArtifactDescriptor
```

Comment:
Not really, it just makes the hierarchy clearer and would later allow to have domain specific properties easily.